tests/matrix_free/face_to_cell_index_nodal_simplex.cc

```diff
@@ -0,0 +1,76 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2020 - 2024 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+
+
+// test the correctness of face_to_cell_index_nodal
+// in internal::MatrixFreeFunctions::ShapeInfo
+// for simplex elements
+
+#include <deal.II/base/quadrature.h>
+#include <deal.II/base/quadrature_lib.h>
+
+#include <deal.II/fe/fe_simplex_p.h>
+#include <deal.II/fe/fe_system.h>
+
+#include <deal.II/matrix_free/shape_info.h>
+
+#include <iostream>
+
+#include "../tests.h"
+
+
+
+template <int dim>
+void
+test(const FiniteElement<dim> &fe, const Quadrature<dim> &quad)
+{
+  for (unsigned int i = 0; i < fe.n_base_elements(); ++i)
+    {
+      internal::MatrixFreeFunctions::ShapeInfo<double> shape_info;
+      shape_info.reinit(quad, fe, i);
+      deallog << "Dim: " << dim << ", degree: " << fe.degree << std::endl;
```

Comment:
```suggestion
      deallog << "Testing: " << fe.get_name() << std::endl;
```
This should print all information, both the dimension, degree and dG/cG